jscomp/bsb/bsb_config_parse.ml

```diff
@@ -271,7 +271,13 @@ let interpret_json
   end;
   let config = 
     {
-      Bsb_config_types.package_name = Bsb_default.get_package_name ();
+      Bsb_config_types.package_name =
+        (match Bsb_default.get_package_name () with
+        | Some name -> name
+        | None ->
+          prerr_endline "Error: Package name is required. Please specify a `name` in `bsconfig.json`";
```

Comment:
As a side note, `begin end` is preferred since there are some traps in  OCaml syntax